File: StandingsApp/StandingsApp/Models/EF/Game.cs
using StandingsApp.Utilities;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace StandingsApp.Models.EF
{
    [Table("Game")]
    public class Game
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int HomeTeamId { get; set; }

        [Required]
        public int AwayTeamId { get; set; }

        public int HomeScore { get; set; }

        public int AwayScore { get; set; }

        public DateTime Scheduled { get; set; }

        public int? WinningTeamId { get; set; }

        public int? LosingTeamId { get; set; }

        [Required, StringLength(15)]
        public string Status { get; set; } = GameStatuses.NEW;

        [StringLength(50)]
        public string? ParkName { get; set; }

        public int? FieldNbr { get; set; }

        [StringLength(125)]
        public string? AddressLine { get; set; }

        [StringLength(50)]
        public string? City { get; set; }

        [StringLength(2)]
        public string? State { get; set; }

        [StringLength(10)]
        public string? Zip { get; set; }

        // navigation properties
        public virtual Team? HomeTeam { get; set; }
        public virtual Team? AwayTeam { get; set; }
    }
}

File: standing-app-ng/src/app/model/game.ts
import { Team } from "./team";

export class Game {
    id: number;
    homeTeamId: number;
    awayTeamId: number;
    homeScore: number;
    awayScore: number;
    scheduled: Date;
    winningTeamId: number;
    losingTeamId: number;
    status: string;
    fieldNbr: number;
    addressLine: string;
    parkName: string;
    city: string;
    state: string;
    zip: string;
    homeTeam: Team;
    awayTeam: Team;

    constructor(
        id: number = 0,
        homeTeamId: number = 0,
        awayTeamId: number = 0,
        homeScore: number = 0,
        awayScore: number = 0,
        scheduled: Date = new Date,
        winningTeamId: number = 0,
        losingTeamId: number = 0,
        status: string = '',
        fieldNbr: number = 0,
        addressLine: string = '',
        city: string = '',
        state: string = '',
        zip: string = '',
        parkName: string = '',
        homeTeam: Team = new Team(),
        awayTeam: Team = new Team()
    ){
        this.id = id;
        this.homeTeamId = homeTeamId;
        this.awayTeamId = awayTeamId;
        this.homeScore = homeScore;
        this.awayScore = awayScore;
        this.scheduled = scheduled;
        this.winningTeamId = winningTeamId;
        this.losingTeamId = losingTeamId;
        this.status = status;
        this.homeTeam = homeTeam;
        this.awayTeam = awayTeam;
        this.fieldNbr = fieldNbr;
        this.addressLine = addressLine;
        this.city = city;
        this.state = state;
        this.zip = zip;
        this.parkName = parkName;
    }
}